### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/deployment/concepts.md`

### PR DESCRIPTION
**Pull request by marcelomarkus**
**- Portuguese translation**
docs/pt/docs/deployment/concepts.md

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

File: docs/pt/docs/deployment/concepts.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+* Segurança - HTTPS
+* Executando na inicialização
+* Reinicializações
+* Replicação (o número de processos em execução)
+* Memória
+* Etapas anteriores antes de iniciar
+
+Veremos como eles afetariam as **implantações**.
+
+No final, o objetivo final é ser capaz de **atender seus clientes de API** de uma forma **segura**, **evitar interrupções** e usar os **recursos de computação** (por exemplo, servidores remotos/máquinas virtuais) da forma mais eficiente possível. 🚀
```
Comment (ceb10n) [inline docs/pt/docs/deployment/concepts.md:16]: I would change:

"No final, o objetivo final..."

for something like:

"No final, o principal objetivo..."

It's odd to use final two times in a row

File: docs/pt/docs/deployment/concepts.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+* Docker
+* Kubernetes
+* Docker Compose
+* Docker em Modo Swarm
+* Systemd
+* Supervisor
+* Gerenciado internamente por um provedor de nuvem como parte de seus serviços
+* Outros...
+
+I'll give you more concrete examples in the next chapters.
```
Comment (ceb10n) [inline docs/pt/docs/deployment/concepts.md:126]: You missed this line. Still needs the translation

File: docs/pt/docs/deployment/concepts.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+
+Mas em muitos casos, você desejará executar vários processos de trabalho ao mesmo tempo.
+
+### Processos Múltiplos - Trabalhadores
+
+Se você tiver mais clientes do que um único processo pode manipular (por exemplo, se a máquina virtual não for muito grande) e tiver **vários núcleos** na CPU do servidor, você poderá ter **vários processos** em execução com o mesmo aplicativo ao mesmo tempo e distribuir todas as solicitações entre eles.
+
+Quando você executa **vários processos** do mesmo programa de API, eles são comumente chamados de **trabalhadores**.
+
+### Trabalhador Processos e Portas
```
Comment (ceb10n) [inline docs/pt/docs/deployment/concepts.md:191]: I would change this title to:

"Processos do Trabalhador e Portas"

**Review by ceb10n — CHANGES_REQUESTED** [review]
Great work 🚀 

You missed a line (still needs translation), and I made a few suggestions too.

✨

**Review by ceb10n — APPROVED** [review]
✨

**Review by cassiobotaro — APPROVED** [review]

**Comment by alejsdev** [comment]
Thanks for one more contribution! @marcelomarkus :sunglasses:

**Pull request merged**